Tests/SwiftAlgorithmsTests/CompactedTests.swift

```diff
@@ -0,0 +1,54 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Algorithms open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import Algorithms
+
+final class CompactedTests: XCTestCase {
+
+  var tests: [[Int?]] = [
+    [],
+    [0],
+    [nil],
+    [0, nil],
+    [nil, 0],
+    [0, nil, 1, nil, 2, nil],
+    [0, 1, 2, nil, nil, nil],
+    [nil, nil, nil, 0, 1, 2],
```

Comment:
Can we use `[nil, nil, nil, 0, 1, 2].uniquePermutations(ofCount: 0...)` here 😄?

Reply:
Fixed! :)